community/en/census/tf20_census.ipynb

```diff
@@ -0,0 +1,1422 @@
+      },
+      "cell_type": "code",
+      "source": [
+        "# Split features and label. We will pass features only to Serving model.\n",
+        "_predict_x, predict_y = predict, predict.pop(_LABEL_COLUMN)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "MNAonpCAm6-N",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Concat training and test data to perform standarization.\n",
+        "all_data = pd.concat([_train_x, _predict_x], keys=[0, 1])\n",
+        "# Standarize predictions using training data + prediction.\n",
+        "standardized_data = standardization(all_data)\n",
+        "train_x, predict_x = standardized_data.xs(0), standardized_data.xs(1)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "9K-iHSMzVubl",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "predict_x.head()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "IpJRFREOPeTo",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Local Predictions"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "1dPMi4g7KbBQ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Predict using Keras\n",
+        "predict_x.to_csv('predictions.csv', header=False)\n",
+        "! more predictions.csv\n",
+        "\n",
+        "predictions = keras_model.predict_classes(predict_x, verbose=1)\n",
+        "print(['<=50K' if x==0 else '>=50K' for x in predictions])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "z9hJPU5YHcLT",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Copyright 2019 The TensorFlow authors\n",
+        "\n",
+        "Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "you may not use this file except in compliance with the License.\n",
+        "You may obtain a copy of the License at\n",
+        "\n",
+        "    http://www.apache.org/licenses/LICENSE-2.0\n",
+        "\n",
+        "Unless required by applicable law or agreed to in writing, software\n",
+        "distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "See the License for the specific language governing permissions and\n",
+        "limitations under the License.\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "K0UXLWaBJnrY",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Questions? Feedback?\n",
```

Comment:
We were discussing a standard way to show the original author at the top of the doc. This should perhaps go there.